package.json

```diff
@@ -13,8 +13,9 @@
     ]
   },
   "scripts": {
-    "build": "yarn build:ui:admin",
-    "build:ui:admin": "cd ui/admin && yarn build",
```

Comment:
Deleting `build:ui:admin` because I think is unnecessary having `build` as default map to `build:ui:admin:oss`. But happy to discuss it.